sys-boot/limine/limine-8.0.8.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="The limine bootloader"
+DESCRIPTION=" Limine is an advanced, portable, multiprotocol bootloader that supports Linux, \
+multiboot1 and 2, the native Limine boot protocol, and more. "
+HOMEPAGE="https://limine-bootloader.org/"
+SOURCE_URI="https://github.com/limine-bootloader/limine/releases/download/v$portVersion/limine-$portVersion.tar.xz"
+REVISION="1"
+LICENSE="BSD (2-clause)"
+COPYRIGHT="2019-2024 mintsuki and contributors"
+CHECKSUM_SHA256="b9ea1a513d1e7c0095c905efc688fc48666b62d27800f3a1a279778a5f4f260a"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	limine$secondaryArchSuffix = $portVersion
+	cmd:limine
```

Comment:
No `$commandBinDir` used, so this would end up in `$prefix/bin/x86` on 32bit if not changed, so needs `$secondaryArchSuffix` for `cmd:limine`

Reply:
Didn't know that, fixed, thanks